src/components/button.css

```diff
@@ -0,0 +1,8 @@
+.button{
+    background: none;
+    border-radius:5px;
+    font-weight:800;
+    height:35px;
+    width:120px;
```

Comment:
The size of elements should be flexible. Imagine that we want to reuse this button with other texts — bigger or smaller — these sizes might not work. What you need:

- use min-width instead of width - so that it grows horizontally. You can even add `max-width`
- use min-height instead of height - so that it grows if needed
- The padding is also missing in case the text is bigger than the current one.

Reply:
These are still used with fixed units. Have you commit your changes?.